### PR TITLE
Adding Cloud integration links to the integration page

### PR DESCRIPTION
**Pull request by ajjindal**
Adding Cloud integration links to the integrations page. This will help us with GCP partnership.

### CODE REVIEW

File: src/docs/product/integrations/gcp-cloud-run/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Google Cloud Run
+
+Cloud Run helps developers save time in building and deploying their applications. Sentry helps save time in resolving production issues by providing detailed debugging capabilities.
+
+### Setup
```
Comment (PeloWriter) [inline src/docs/product/integrations/gcp-cloud-run/index.mdx:5]:
```suggestion
### Set Up
```

File: src/docs/product/integrations/gcp-cloud-run/index.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+1. [Create an account](https://sentry.io/signup/) (or [Login](https://sentry.io/auth/login/)) on Sentry
+2. Choose your application platform
+
+    ![Sentry Choose Platform](sentry-choose-platform.png)
+
+3. Follow instructions to instrument your Cloud Run application. Detailed platform docs [here](http://docs.sentry.io/).
```
Comment (PeloWriter) [inline src/docs/product/integrations/gcp-cloud-run/index.mdx:12]:
```suggestion
3. Follow instructions to instrument your Cloud Run application. Detailed platform docs [here](http://docs.sentry.io/platforms/).
```

File: src/docs/product/integrations/gcp-cloud-run/index.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+4. Deploy your Cloud Run application with GCP.
+
+That's it! Sentry will now report all issues from your Cloud Run application.
```
Comment (PeloWriter) [inline src/docs/product/integrations/gcp-cloud-run/index.mdx:15]: If you would like, you can use this standard verbiage "Once this is done, all unhandled exceptions are automatically captured by Sentry."

**Review by PeloWriter — APPROVED** [review]
LGTM. 

@MimiDumpling - heads up, this PR may create a merge conflict for https://github.com/getsentry/sentry-docs/pull/2294

File: src/docs/product/integrations/gcp-cloud-run/index.mdx (hunk 3 above)
Comment (ajjindal) [reply src/docs/product/integrations/gcp-cloud-run/index.mdx:15]: I will leave it as it is since we will capture transactions a well.

**Pull request merged**